README.md

```diff
@@ -1,71 +1,107 @@
-# rex
+# REX
 
-rex is a Really EXcellent way to develop webapps.
+REX is a Really EXcellent way to develop web-apps.
 
-# Quickstart
+## Quick start
 
-First, (install Docker)[https://docs.docker.com/install/].
+First, [install Docker](https://docs.docker.com/install/).
 
 Next, run
 
-```
-npm run db:sync-force
-npm start
+```bash
+$ npm run docker:db:sync:force
```

Comment:
when you copy/paste the text in the quoted block here will it include the `$` ? if so I'd like to keep this the way it is (though the `bash` above is great) because I'd like it to be possible to cut and paste the entire block into a shell without an intermediate editing step